src/components/expanded-state/swap-settings/MaxToleranceInput.tsx

```diff
@@ -0,0 +1,161 @@
+import lang from 'i18n-js';
+import React, {
+  forwardRef,
+  useCallback,
+  useEffect,
+  useImperativeHandle,
+  useRef,
+  useState,
+} from 'react';
+import { InteractionManager } from 'react-native';
+import { useSelector } from 'react-redux';
+import { Icon } from '../../icons';
+import { Text } from '../../text';
+import StepButtonInput from './StepButtonInput';
+import {
+  Box,
+  Column,
+  Columns,
+  Stack,
+  Text as StyledText,
```

Comment:
why the renaming here?

Reply:
ah i see it https://github.com/rainbow-me/rainbow/pull/3512/files#diff-02f777ddf40c7715a0d84e2de0dd8f7bd40d61cc28bab0f37dbdd1446a88c91dR134